lib/bashly/commands/generate.rb

```diff
@@ -116,7 +118,7 @@ def master_script_path
       end
 
       def config
-        @config ||= Config.new "#{Settings.source_dir}/bashly.yml"
+        @config ||= Config.new("#{Settings.source_dir}/bashly.yml").compose
```

Comment:
This is where the composition happens, by just calling the `#compose` on the refined Hash, it will convert any `import: path` to the contents of the YAML represented by the path.